docker/openpose/Dockerfile

```diff
@@ -6,39 +6,49 @@ WORKDIR /workspace
 
 ARG DEBIAN_FRONTEND=noninteractive
 
-# Install dependencies
-RUN apt-get update && apt-get install -y \
-    build-essential \
-    cmake \
-    git \
-    wget \
-    libatlas-base-dev \
-    libprotobuf-dev \
-    libleveldb-dev \
-    libsnappy-dev \
-    libhdf5-serial-dev \
-    protobuf-compiler \
-    libboost-all-dev \
-    libgflags-dev \
-    libgoogle-glog-dev \
-    liblmdb-dev \
-    python3-dev \
-    python3-pip \
-    python-is-python3 \
-    libopencv-dev \
-    python3-opencv \
+ENV TORCH_CUDA_ARCH_LIST="7.0;7.5;8.0;8.6+PTX"
```

Comment:
Thanks for updating this, it appears that I have totally missed that when I optimized the docker files :smile: Hope it didn't cause bigger inconveniences

Reply:
No problem! Previously I tried to build MaskAnyone from scratch and got an error, but didn't look into it again until working on this api container. Then it was fixed relatively quickly.